### ISSUE TITLE
Able to config FROM e-mail field 

### ISSUE THREAD

**Issue by rbukovansky** [gogits/gogs#4922]
- Gogs version (or commit ref): 0.11.34.1122
- Git version: 1.7.1
- Operating system: CentOS 6
- Database (use `[x]`):
  - [x] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gogs.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist (usually found in `log/gogs.log`):

## Description

When you are going to make SUBJECT parameter of mailer in app.ini working (see #4178), can we please get possibility of configuring FROM field of e-mails too? Thank you.

**Comment by Fastidious** [comment]
@rbukovansky you mean, [like this](https://github.com/gogits/gogs/blob/6f2347fc71f17b5703a9b1f383a2d3451f88b741/conf/app.ini#L245)?

**Comment by rbukovansky** [comment]
@Fastidious Exactly. It's not working for me at all... Still the e-mails are coming from
```
"<name_of_user surname_of_user>" <gogs@example.com>
```
 even I do have
```
FROM = `"Gogs" <gogs@example.com>`
```
set.

**Comment by rbukovansky** [comment]
BTW wouldn't mind using some variables like `{$APP_NAME} {$USER.NAME} {$USER.LASTNAME}` etc. in those two field either...

**Comment by Unknwon** [comment]
Different kinds of emails have different kinds of format.
